File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int arr11 = 0;  //배열 1 원소들의 합
        int arr22 = 0;  //배열 2 원소들의 합
        if (arr1.length < arr2.length) {
            answer = -1;
        }
        else if (arr1.length > arr2.length) {
            answer = 1;
        }
        else if (arr1.length == arr2.length) {
            for (int i = 0; i < arr1.length; i++) {
                arr11 += arr1[i];
                arr22 += arr2[i];
            }
            if (arr11 > arr22) {
                answer = 1;
            }
            else if (arr11 < arr22) {
                answer = -1;
            }
            else if (arr11 == arr22) {
                answer = 0;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    num1 = 0
    num2 = 0
    for i in range(len(arr1)) :
        num1 += arr1[i]
    for j in range(len(arr2)) :
        num2 += arr2[j]
    if len(arr1) > len(arr2) :
        answer = 1
    elif len(arr1) < len(arr2) :
        answer = -1
    else :
        if num1 > num2 : answer = 1
        elif num1 < num2 : answer = -1
        else : answer = 0
    return answer